voicebase-client-ruby.gemspec

```diff
@@ -19,11 +19,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.11"
-  spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 3.0"
-  spec.add_development_dependency "timecop", "~> 0.8"
+  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_development_dependency "rake", "~> 12.1"
+  spec.add_development_dependency "rspec", "~> 3.7"
+  spec.add_development_dependency "timecop", "~> 0.9"
 
-  spec.add_dependency "httparty"
-  spec.add_dependency "activesupport", "~> 4"
+  spec.add_dependency "httparty", "~> 0.15"
+  spec.add_dependency "activesupport", "~> 5.1"
```

Comment:
Can we just make this ">= 4" so that it will work with both Rails 4 and Rails 5. Do we know what we use that depends on activesupport? Any chance we can remove this dependency?

Reply:
Actually looking at it. we don't need this at all... `~> 4` is the same as `>= 4` since the top level is allowed to vary.

Also we shouldn't add a restriction on gems in a library unless these is a known incompatibility, so we don't cause unnecessary restrictions for users of the library

Reply:
@aselder I'd like to get your advice on how to proceed. I wasn't sure last week and so it occurred to me that if I upgrade the Rails gem to 5+ that ought to do it. 

It sounds like you're recommending I undo 100% of the changes to the gem versions.

Given that, what work is there to do? I don't know how to verify Rails 5 compatibility.

Reply:
@jhogsett I think @aselder is recommending removing unnecessary restrictions -- looks like we originally were over-specifying gem versions, which we can either relax or remove...